transbank/webpay/webpayplus/exceptions/transbank_exception.py

```diff
@@ -0,0 +1,10 @@
+
+class TransbankException(Exception):
+    DEFAULT_MESSAGE = 'An error has happened, verify given parameters and try again.'
```

Comment:
This message should be more generic, it would point the integrator into looking for errors in their code when the exception something completely different.

You can try something similar to `An unidentified error has occurred.` and try to pass the stacktrace so the integrator has information to report back to us